src/dotnet/ProjectRootElementExtensions.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Build.Construction;
+
+namespace Microsoft.DotNet.Tools.Common
+{
+    public static class ProjectRootElementExtensions
+    {
+        public static string GetProjectTypeGuid(this ProjectRootElement rootElement)
+        {
+            return rootElement
+                .Properties
+                .FirstOrDefault(p => p.Name == "ProjectTypeGuids")
```

Comment:
Case insensitive.

Reply:
Thanks, I'll fix this and the other places we're reading properties in a case-sensitive manner.